### DISCUSSION TITLE
Upgrade to 4.3.6 error contrib/apache.conf

### DISCUSSION THREAD

**Discussion by darrenwoods3** [netbox-community/netbox#12055] [Help Wanted!]
when upgrading from 3.4.5 to 3.4.6 and i run sudo git pull origin master, i get the following error:
error: Your local changes to the following files would be overwritten by merge:
        contrib/apache.conf
Please commit your changes or stash them before you merge.
Aborting

**Comment by candlerb** [comment]
It's saying that you have changed contrib/apache.conf within your checked-out copy of the repository, so it's now different to the upstream version.

I suggest you take a copy of your modified file, just in case:

```
sudo cp contrib/apache.conf contrib/apache.conf.mine
```

and then revert it to the official version:

```
sudo git checkout contrib/apache.conf
```

After that your "git pull" should work.
